### PR TITLE
Update Slovak translation

### PR DESCRIPTION
**Pull request by Roxterat**
An update to sk.po file after it's first appearance.

### CODE REVIEW

**Comment by Roxterat** [comment]
Rebased and ready for review @zenseii

File: files/lang/sk.po [hunk 1]
```diff
@@ -100,531 +100,579 @@ msgid "X-L"
...
-msgstr ""
+msgstr "Vyp"
 
 msgid "On"
-msgstr ""
+msgstr "Zap"
 
 msgid "Auto Spell Casting"
-msgstr ""
+msgstr "Použi kúzla v bitve"
```
Comment (joyinko) [inline files/lang/sk.po:381]: "Automatické kúzlenie"

File: files/lang/sk.po [hunk 2]
```diff
@@ -100,531 +100,579 @@ msgid "X-L"
...
-msgstr ""
+msgstr "Ukáž kurzor"
 
 msgid "Audio"
-msgstr ""
+msgstr "Zvuk"
 
 msgid "Settings"
-msgstr ""
+msgstr "Nastavenie"
```
Comment (joyinko) [inline files/lang/sk.po:396]: "Nastavenia"

File: files/lang/sk.po [hunk 3]
```diff
@@ -100,531 +100,579 @@ msgid "X-L"
...
 
 msgid "Configure"
-msgstr ""
+msgstr "Konfigurácia"
```
Comment (joyinko) [inline files/lang/sk.po:399]: "Konfiguruj"

File: files/lang/sk.po [hunk 4]
```diff
@@ -100,531 +100,579 @@ msgid "X-L"
...
-msgstr ""
+msgstr "Obrana"
 
 msgid "Spell Power"
-msgstr ""
+msgstr "Sila kúziel"
 
 msgid "Knowledge"
-msgstr ""
+msgstr "Vedomosť"
```
Comment (joyinko) [inline files/lang/sk.po:521]: "Vedomosti"

**Comment by joyinko** [comment]
@Roxterat prosím pozri na moje návrhy. Please take a look, I left few suggestions.

File: files/lang/sk.po [hunk 5]
```diff
@@ -100,531 +100,579 @@ msgid "X-L"
...
 "EXPANSION\n"
 "CAMPAIGN"
 msgstr ""
+"ROZŠÍRENÁ\n"
+"KAMPAŇ"
 
 msgid "HOT SEAT"
 msgstr ""
+"HORÚCA\n"
+"PÔDA"
```
[the remark this answers is not in the document]
Comment (Roxterat) [reply files/lang/sk.po]: Applied.

File: files/lang/sk.po (hunk 1 above)
Comment (Roxterat) [reply files/lang/sk.po]: Applied.

File: files/lang/sk.po (hunk 2 above)
Comment (Roxterat) [reply files/lang/sk.po]: Applied.

File: files/lang/sk.po (hunk 3 above)
Comment (Roxterat) [reply files/lang/sk.po]: Applied.

**Review by joyinko — APPROVED** [review]

**Comment by Roxterat** [comment]
@zenseii conflickts resolved and the current file content should correspond with the latest file structure in project's master unless more changes would appear. Ready for review.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Roxterat , thank you very much for this translation update!

**Pull request merged**